vault/dendron.tutorial.rich-formatting.md

```diff
@@ -60,13 +59,62 @@ sequenceDiagram
     Bob-->>John: Jolly good!
 ```
 
+#### Code Blocks
+
+Markdown provides a way to render syntax highlighting and formatting of code.
+
+```python
+# Whoa, I'm doing math in Python
+result = 2 + 2
+print(result)
+```
+
+The above code is in python. Both the editor view and preview view should show some syntactical flavor to the code block. You can also have code blocks with or without designated language markers.
+
+```bash
+# Whoa, this is a code block explicitly
+## stating it is in bash!
+echo "This is a bash example!"
+ps -ef | grep codium
+```
+
+Notice how `bash` is used on the first line of the code block, and `python` on the previous code block. If missing a language marker, code blocks look like this:
+
+```
+# Whoa, this is still bash, but without
+## it explicitly stated on the first line!
+echo "This is a bash example!"
+ps -ef | grep codium
+```
+
 ### Note References
 
 You can link to a specific section of a different note and have the content in-lined into the current note.
 
-![[Join the community of Dendrologists|dendron.tutorial.conclusion#join-the-community-of-dendrologists:#^discord-block-ref]]
+> 🌱 **ACTION:** Create [[recipes.favorites]], if you haven't yet. We're going to add some content in here to reuse as a note reference!
+
+Add the following contents to [[recipes.favorites]]:
+
+```markdown
+My favorite recipes, by far. I must avoid.
+
+- [[recipes.italian.desserts.tiramisu]]
+- [[Chinese Wontons|recipes.chinese.appetizers.wontons]]
+```
+
+In [[recipes]], add the following contents:
+
+```markdown
+## Favorites
+
+![[recipes.favorites]]
+```
+
+Take a look in the preview view to see how this renders:
+
+![[recipes.favorites]]
```

Comment:
heads up, this doesn't exist and won't exist on the website. should be a screenshot

Reply:
> heads up, this doesn't exist and won't exist on the website. should be a screenshot

I've been wondering how to provide directions to notes that don't exist, since the user is meant to create them in their new workspace during the tutorial. That means we have several wikilinks, and things like this note ref, that are a reference to nowhere since they are meant to be interactive. Should I just say something like:

```markdown
Remove the surrounding backticks from the following, then look in the preview view to see how this renders:

`![[recipes.favorites]]`
```

Reply:
I'll give that a shot, and do so with all of the others. Because they otherwise will appear as broken links in the published results, or will appear in any `findBrokenLinks` calls.